starfish/imagestack/physical_coordinate_calculator.py

```diff
@@ -121,11 +123,12 @@ def _pixel_offset_to_physical_coordinate(physical_pixel_size: Number,
     return coordinates_at_pixel_offset_0
 
 
-def _recalculate_physical_coordinate_range(coord_min: float,
-                                           coord_max: float,
-                                           dimension_size: int,
-                                           indexer
-                                           ) -> Tuple[Number, Number]:
+def _recalculate_physical_coordinate_range(
+        coord_min: float,
+        coord_max: float,
+        dimension_size: int,
+        indexer
+) -> Tuple[Number, Number]:
```

Comment:
So if there's listed params like this I should always start them on the next line?

Reply:
So my preference is that for multiline params (calls and method declarations), that we start on the next line (basically the second option of https://www.python.org/dev/peps/pep-0008/#indentation).

Rationale: If the method name changes, we do not need to change the indentation for every line.